pkg/utils/has/controller.go

```diff
@@ -92,7 +92,13 @@ func (h *SuiteController) DeleteHasComponent(name string, namespace string) erro
 }
 
 // CreateComponent create an has component from a given name, namespace, application, devfile and a container image
-func (h *SuiteController) CreateComponent(applicationName, componentName, namespace, gitSourceURL, containerImageSource, outputContainerImage string) (*appservice.Component, error) {
+func (h *SuiteController) CreateComponent(applicationName, componentName, namespace, gitSourceURL, containerImageSource, outputContainerImage, secret string) (*appservice.Component, error) {
```

Comment:
If this change is based on the [change in the HAS api schema](https://github.com/redhat-appstudio/application-service/blob/86571e38f52e6bbe67b945dd58b695af26ae834d/config/crd/bases/appstudio.redhat.com_components.yaml#L54), I think it'd make sense to remove the `outputContainerImage` parameter, wdyt?

Reply:
Yeah, that's a good point. I'll fix that.